conf/defaults.ini

```diff
@@ -1189,6 +1189,9 @@ cloudWatchDynamicLabels = true
 # New expandable navigation
 newNavigation = true
 
+# Prometheus Streaming JSON Parser
+prometheusStreamingJSONParser = true
```

Comment:
rather than here, set the default expression in https://github.com/grafana/grafana/blob/main/pkg/services/featuremgmt/registry.go#L96